src/libs/actions/Report.ts

```diff
@@ -3499,7 +3499,7 @@ function prepareOnboardingOptimisticData(
     const {reportID: targetChatReportID = '', policyID: targetChatPolicyID = ''} = targetChatReport ?? {};
 
     // Introductory message
-    const introductionComment = ReportUtils.buildOptimisticAddCommentReportAction(CONST.ONBOARDING_INTRODUCTION, undefined, actorAccountID);
+    const introductionComment = ReportUtils.buildOptimisticAddCommentReportAction(undefined, actorAccountID);
```

Comment:
Yeah this is not gonna work. You'll need to remove the `introductionComment` variable and anything else referencing it.

Reply:
In this case, it looks like that would be `introductionCommentAction` and also `introductionMessage`. So, you'd need to remove anything referencing them too. Which looks like just:

https://github.com/Expensify/App/blob/c701612031a0f022cfc7e277ebff5b315420d8d6/src/libs/actions/Report.ts#L3870

https://github.com/Expensify/App/blob/c701612031a0f022cfc7e277ebff5b315420d8d6/src/libs/actions/Report.ts#L3756

https://github.com/Expensify/App/blob/c701612031a0f022cfc7e277ebff5b315420d8d6/src/libs/actions/Report.ts#L3779

Reply:
OK did that work? Thanks for the help!

Reply:
We gotta get rid of this line too.

```suggestion
```